### ISSUE TITLE
Pass input profiles in text file to inStrain compare

### ISSUE THREAD

**Issue by jdwinkler-lanzatech** [MrOlm/inStrain#94]
Hi,

I might be processing a fairly large number of samples via inStrain compare soon, and I suspect I'll hit the limit on the length of the CLI argument used if I go through bash or something similar. Is it possible to add a CLI option containing a list of input folders (one folder per line) to avoid that issue? Thanks for the help again!

**Comment by MrOlm** [comment]
Yeah this is a great feature request- I'll try and add it in the next inStrain update. I'll note that after ~100 profiles or so things can get really slow because profiles are compared in a pair-wise manner. I'm working on a greedy clustering algorithm to get around this, but it won't be ready any time soon.

Best,
Matt

**Comment by jdwinkler-lanzatech** [comment]
Yeah, that'll probably put a damper on my analysis then. Thanks for the head's up!

**Comment by kimndlovu** [comment]
Hello @MrOlm 
I am having the same issue with 159 samples and inStrain compare keeps returning an OOM error with 475GB memory given.

profiles=$(ls -d ${IS_out}/*.IS) 
inStrain compare -i ${profiles} -o allsamples.fa.IS.COMPARE

 I tried running the greedy_clustering (v.1.3.9) and it says greedy clustering not available. 

inStrain compare -i ${profiles} -o allsamples.fa.IS.COMPARE --greedy_clustering --g_ani 0.99 --g_cov 0.99 --g_mm 100


any advice on how to compare these 159 samples? WHat is the minimum memory to allocate for these?

Thanks,
kim
